### PR TITLE
[plugins/enocean]: Rework of Eltako Shutter Actor FSB71

### PR DESCRIPTION
**Pull request by hasenradball**
## Updapte of eep_parser
- Rename of profile F6_02_03_01 -> F6_0G_03
- add POSITION
- Add new profile A5_0G_03

## Update of Readme.md
- update due to changes above

### CODE REVIEW

**Comment by aschwith** [comment]
Nice. Please 
a) Increase version number in init.py and plugin.yaml as well
b) Add device name (e.g. Eltako XYZ) for all custom EEPs in readme.conf.

That's it. Thanks

**Comment by hasenradball** [comment]
Hi,

thanks for your feedback.
I hope now erverything is fine.

:-)

File: enocean/eep_parser.py [hunk 1]
```diff
@@ -262,6 +262,29 @@ def _parse_eep_A5_3F_7F(self, payload, status):
...
 
+    def _parse_eep_A5_0G_03(self, payload, status):
+        '''
+        4 Byte communication(4BS) Telegramm
+        Status command from bidirectional shutter actors:
+            Eltako FSB61NP-230V, FSB71, FJ62/12-36V DC, FJ62NP-230V
+        If actor is stopped during run it sends the feedback of runtime and the direction
+        This enables to calculate the actual shutter position
+        Key Description:
+            MOVE: runtime of movement in s (with direction: "+" = up; "-" = down)
```
Comment (hasenradball) [inline enocean/eep_parser.py:273]: Hab hier noch einen Typo gefunden:
"-" : bedeutet hoch fahren
"+": bedeutet runterfahren

Comment (hasenradball) [reply enocean/eep_parser.py:273]: Werde ich noch nach schieben...

**Pull request merged**